client/layout/guided-tours/tours/checklist-user-avatar-tour.js

```diff
@@ -0,0 +1,81 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { translate } from 'i18n-calypso';
+import { noop } from 'lodash';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ButtonRow,
+	Continue,
+	makeTour,
+	SiteLink,
+	Step,
+	Tour,
+} from 'layout/guided-tours/config-elements';
+
+export const ChecklistUserAvatarTour = makeTour(
+	<Tour name="checklistUserAvatar" version="20171205" path="/non-existent-route" when={ noop }>
+		<Step
+			name="init"
+			target="edit-gravatar"
+			placement="beside"
+			arrow="left-top"
+			style={ {
+				animationDelay: '0.7s',
+			} }
+		>
+			<p>
+				{ translate(
+					'Personalize your posts and comments with your Avatar. ' +
+						'Click on this image to upload a new profile picture.'
+				) }
+			</p>
+			<ButtonRow>
+				<Continue target="edit-gravatar" step="click-done" click hidden />
+				<SiteLink isButton={ false } href="/checklist/:site">
+					{ translate( 'Return to the checklist' ) }
+				</SiteLink>
+			</ButtonRow>
+		</Step>
+
+		<Step
+			name="click-done"
+			target="image-editor-button-done"
+			arrow="bottom-center"
+			placement="above"
+		>
+			<Continue target="image_editor_button_done" step="finish" click>
+				{ translate(
+					"Let's make sure it looks right before you press {{b}}Change My Photo{{/b}} to save your changes.",
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Let's make sure it looks right before you press {{b}}Done{{/b}} to save your changes.' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).